core/src/main/java/io/grpc/internal/DnsNameResolver.java

```diff
@@ -132,10 +132,11 @@
     this.executorResource = executorResource;
     // Must prepend a "//" to the name when constructing a URI, otherwise it will be treated as an
     // opaque URI, thus the authority and host of the resulted URI would be null.
-    URI nameUri = URI.create("//" + name);
+    URI nameUri = URI.create("//" + checkNotNull(name));
```

Comment:
Pass a second argument to checkNotNull, so it is easier to figure out which value was NPE. So I'd suggest passing `"name"` here.

Reply:
thanks, done.